config/mfDefs.gem

```diff
@@ -43,8 +43,8 @@ UserDynamicLinkFlags =
 IncludeDirs+=$(XDAQ_ROOT)/include
 
 ## Python
-PYTHON_VERSION ?= $(shell python -c "import distutils.sysconfig;print distutils.sysconfig.get_python_version()")
-PYTHON_INCLUDE_PREFIX ?= $(shell python -c "import distutils.sysconfig;print distutils.sysconfig.get_python_inc()")
+PYTHON_VERSION ?= $(shell python2.7 -c "import distutils.sysconfig;print distutils.sysconfig.get_python_version()")
+PYTHON_INCLUDE_PREFIX ?= $(shell python2.7 -c "import distutils.sysconfig;print distutils.sysconfig.get_python_inc()")
```

Comment:
SCL `python27` has been installed at p5, so I would prefer reverting this, and setting the desired `python` version in some setup script, to having this fixed to `python2.7`